pkg/console/subresource/deployment/deployment.go

```diff
@@ -64,10 +66,8 @@ type volumeConfig struct {
 }
 
 func DefaultDeployment(operatorConfig *operatorv1.Console, cm *corev1.ConfigMap, serviceCAConfigMap *corev1.ConfigMap, defaultIngressCertConfigMap *corev1.ConfigMap, trustedCAConfigMap *corev1.ConfigMap, sec *corev1.Secret, proxyConfig *configv1.Proxy, infrastructureConfig *configv1.Infrastructure, canMountCustomLogo bool) *appsv1.Deployment {
-	labels := util.LabelsForConsole()
-	meta := util.SharedMeta()
-	meta.Labels = labels
-	deploymentAnnotations := map[string]string{
+	deployment := resourceread.ReadDeploymentV1OrDie(assets.MustAsset("deployments/console-default-deployment.yaml"))
```

Comment:
@jhadvig I'm not sure if we will run into the RBAC issue here as these manifests were not managed by the CVO AFAIK. Can you confirm?

Reply:
we should not run into any RBAC issues here, since its handled by the console-operator itself and it does have proper RBAC for it.